galaxy/api/internal/tests/test_collection_views.py

```diff
@@ -56,3 +62,25 @@ def test_get_collections2(self):
         assert collection['namespace'] == self.namespace.id
         assert collection['name'] == self.collection.name
         assert collection['latest_version']['version'] == self.version.version
+
+    def test_get_collection_details(self):
+        url = "/api/internal/ui/collections/mynamespace/mycollection/"
+        resp = self.client.get(url)
+
+        collection = resp.json()
+
+        assert collection['namespace']['id'] == self.namespace.id
+        assert collection['namespace']['name'] == self.namespace.name
+        assert collection['name'] == self.collection.name
+        assert collection['latest_version']['version'] == self.version.version
+        assert len(collection['all_versions']) == 2
+        assert 'contents' not in collection['all_versions'][0]
```

Comment:
positive assertions are clear to read "This thing is supposed to be such and such" but _negative_ assertions might be less clear to someone coming and reading the test later. "_Why_ shouldn't a `contents` entry exist here?" Maybe add a comment to clarify?